vm-virtio/src/net_util.rs

```diff
@@ -30,9 +38,31 @@ pub const KILL_EVENT: DeviceEventT = 3;
 pub const PAUSE_EVENT: DeviceEventT = 4;
 // Number of DeviceEventT events supported by this implementation.
 pub const NET_EVENTS_COUNT: usize = 5;
+// The device has been dropped.
+const CTRL_QUEUE_EVENT: DeviceEventT = 0;
+// The device has been dropped.
+const CTRL_KILL_EVENT: DeviceEventT = 1;
+// The device should be paused.
+const CTRL_PAUSE_EVENT: DeviceEventT = 2;
```

Comment:
You should simply reuse the existing `KILL_EVENT` and `PAUSE_EVENT` instead of defining new ones. And about `CTRL_QUEUE_EVENT`, place it at the top, as the number 0, shifting everything else.

Reply:
Reuse KILL_EVENT and PAUSE_EVENT. Regarding CTRL_QUEUE_EVENT, I do not change it since control queue is handled in a separate thread, I think it's better to not list it with the events defined for Tx/Rx handling thread.